specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4286,8 +4342,35 @@
       },
       "enum": [
         "detection_01",
-        "detection_02"
+        "detection_02",
+        "detection_03"
       ]
+    },
+    "returnFaceId": {
+      "name": "returnFaceId",
+      "in": "query",
+      "x-ms-parameter-location": "method",
+      "description": "A value indicating whether the operation should return faceIds of detected faces.",
+      "type": "boolean",
+      "default": true
+    },
+    "returnFaceLandmarks": {
+      "name": "returnFaceLandmarks",
+      "in": "query",
+      "x-ms-parameter-location": "method",
+      "description": "A value indicating whether the operation should return landmarks of the detected faces.",
+      "type": "boolean",
+      "default": false
+    },
+    "faceIdTimeToLive": {
+      "name": "faceIdTimeToLive",
+      "in": "query",
+      "x-ms-parameter-location": "method",
+      "description": "The number of seconds for the face ID being cached. Supported range from 60 seconds up to 86400 seconds. The default value is 86400 (24 hours).",
```

Comment:
probably we should use `faceId` here , to keep consistent

Reply:
fixed